extensions/email/src/main/java/org/mule/extension/email/internal/EmailFlags.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.email.internal;
+
+/**
+ * The {@link EmailFlags} class represents the set of flags on a Message. Flags
+ * are composed of predefined system flags that most folder
+ * implementations are expected to support.
+ *
+ * @since 4.0
+ */
+public class EmailFlags
+{
+
+    public EmailFlags(Boolean answered, Boolean deleted, Boolean draft, Boolean recent, Boolean seen)
```

Comment:
why boxed booleans? can any of these be null? always use native whenever possible